src/js/components/ImgMenuItem.vue

```diff
@@ -1,62 +1,75 @@
 <template>
-  <span class="clickable-menu-item" @click="setRouter">
-    {{institution}}: {{plate}}, {{fragment}} ({{dataId}})
-  </span>
+  <div>
+    <span class="clickable-menu-item" @click="setRouter">
+      {{institution}}: {{plate}}, {{fragment}} ({{dataId}})
+    </span>
+    <div class="children" v-show="open">
+        <ul>
+          <li v-for="child in children">
+            <art-menu-item :data-id="child.artefact_id"></art-menu-item>
+          </li>
+        </ul>
+    </div>
+  </div>
 </template>
 
 <script>
 
 import { mapGetters } from 'vuex'
+import ArtMenuItem from './ArtMenuItem.vue'
 
 export default {
   props: {
     dataId: Number,
     plate: '',
     fragment: '',
     institution: '',
+    versionID: '',
+  },
+  components: {
+    'art-menu-item': ArtMenuItem,
   },
   data() {
     return {
+      children: [],
+      open: false,
     }
   },
   computed: {
     ...mapGetters(['username', 'sessionID', 'userID'])
   },
   methods: {
     fetchChildren() {
-
+      this.children = []
       // we'll lazy load children, but cache them
-      if (this.lastFetch !== this.requestType[this.menuType]) {
-        this.$router.push({ name: 'workbenchScrollVersionPlateFragment',
-                            params: { scrollID: this.scrollID, 
-                                      scrollVersionID: this.versionID }
-        })
-
-        this.$post('resources/cgi-bin/GetImageData.pl', {
-        transaction: this.requestType[this.menuType],
-        combID: this.scrollDataID,
-        user: this.userID,
+      this.$post('resources/cgi-bin/GetImageData.pl', {
+        transaction: 'getArtOfImage',
+        image_id: this.dataId,
         version_id: this.versionID,
         SESSION_ID: this.sessionID
```

Comment:
The `$post` method takes care of appending the `SESSION_ID` to all outgoing requests if it's available in the store so we can remove this line here and elsewhere.